crates/superposition_types/migrations/2025-05-16-082925_experiment_groups/up.sql

```diff
@@ -0,0 +1,13 @@
+-- Your SQL goes here
+CREATE TABLE IF NOT EXISTS public.experiment_groups(
+    hash TEXT PRIMARY KEY,
+    name TEXT NOT NULL,
+    description TEXT NOT NULL,
+    context JSON NOT NULL,
+    traffic_percentage integer NOT NULL CONSTRAINT traffic_percentage_range CHECK (traffic_percentage >= 0 AND traffic_percentage <= 100),
+    experiments_inside_ids TEXT[] NOT NULL,
```

Comment:
`contained_experiment_ids` or `scoped_experiment_ids` or `included_experiment_ids` ?
```suggestion
    contained_experiment_ids TEXT[] NOT NULL,
```

Reply:
I'm thinking `internal_experiment_ids`, does that work?

Reply:
went with member_experiment_ids